docker-stack.yml

```diff
@@ -87,8 +87,9 @@ services:
 
                 if [ $$django_tables_exist -gt 0 ] || [ $$REFRESH_DB == 'true' ]; then
                     ./refresh-data.sh 
-                    ./cfgov/manage.py update_index
                 fi
+
+                ./cfgov/manage.py update_index
```

Comment:
@marcesher is this a deliberate change?

Reply:
@chosak Yes. I added an explanation to the commit but haven't added it to the PR yet. I discovered the bug while testing the notification changes. I will update the PR description with the explanation momentarily

Reply:
@chosak If you'd prefer I pull that into a separate PR, I'm happy to do so. Whatever the team norms are for a case like this, I'll gladly abide

Reply:
I'll defer to others more familiar with the Elasticsearch changes -- I don't know enough about those tests to understand why this would be necessary again. Is it because the ES content gets wiped out with each redeploy, unlike the Postgres content (if `REFRESH_DB` is unset)?

Reply:
The good news is that updating the index will not take long if there are no changes to it (if I remember correctly).

Reply:
Wait 🤦  that's not true at all if the image is rebuilt and there is no index. I do think we need to take a look at that alternate approach sooner, rather than later.

Reply:
Roger that, Scott. Just this morning, we prioritized solving the Jenkins job-dsl rebuild problem that we discussed elsewhere and which I anticipated would be a tough nut to crack, and fortunately we solved that unexpectedly quickly. So that serendipitously leaves an opening for us to prioritize this in our current sprint, as well. I'll talk to the team about it. 🙌 

Would you be OK merging this PR now, knowing that we'll prioritize the alternate approach this sprint?

Reply:
Yeah, for sure! Merge when ready :)